script.js

```diff
@@ -0,0 +1,257 @@
+const products = [
+  {
+    productName: 'bread',
+    productData: {
+      weight: 200,
+      certificate: true,
+      date0fExpiry: '21/10/2022',
+      sugarFree: true,
+    },
+    productProvider: 'FOP Skyba R.R.',
+    productCountry: 'https://www.worldometers.info/img/flags/ar-flag.gif',
```

Comment:
Easier to keep a name for flag. Such as 'UA'. And then based on that filed using template string:

`https://www.worldometers.info/img/flags/${product.productCountry}-flag.gif'`